frontend/src/scenes/surveys/SurveyTemplates.tsx

```diff
@@ -28,7 +28,7 @@ export function SurveyTemplates(): JSX.Element {
                     </LemonButton>
                 }
             />
-            <div className="flex flex-row flex-wrap gap-8 justify-center mt-8">
+            <div className="flex flex-row flex-wrap gap-8 mt-8">
```

Comment:
Should we not be centering here?

Reply:
My change is correct.

Here is centered:

<img width="1441" alt="image" src="https://github.com/PostHog/posthog/assets/154479/1decdbc4-c821-4942-8390-3d90f8a5d86a">

This is not centered (more like a grid): 

<img width="1442" alt="image" src="https://github.com/PostHog/posthog/assets/154479/dfe14037-58dd-4751-8395-d6600cbb7c04">